packages/language-typescript/src/index.ts

```diff
@@ -87,10 +87,20 @@ ${interfaces}
 const fixDescriptionDocblock: (description?: string) => string | undefined = description =>
   description ? description.replace(/\n/g, '\n* ') : description;
 
-export const DEFAULT_DOCUMENTATION_GENERATOR: GenerateDocumentation = ({ description, tags = [] }) => (description || tags.length) ? `
+export const DEFAULT_DOCUMENTATION_GENERATOR: GenerateDocumentation = ({ description, tags = [] }) => {
+  if (!description && !tags.length) {
+    return '';
+  }
+  const arr: Array<string | undefined> = [
+    fixDescriptionDocblock(description),
+    ...tags.map(({ tag, value }) =>
+      `@${tag} ${typeof value === 'object' ? JSON.stringify(value) : value}`)
```

Comment:
i think we should just `JSON.stringify` everything, that way strings are quoted as well. What do you think?

Reply:
I'm pretty indifferent about it. I guess the main disadvantage with it is that it all looks like strings strings now (except objects), even numbers.